plugins/com.google.cloud.tools.eclipse.javax.annotations/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/org.jsr-305_3.0.1.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
```

Comment:
I'm using JavaSE-1.7, so I think this is in conflict. (The [README.md](https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/blob/master/README.md) says to use JavaSE-1.7.)